docs/components/concepts/process-instance-migration.md

```diff
@@ -0,0 +1,206 @@
+---
+id: process-instance-migration
+title: "Process instance migration"
+description: "Use process instance migration to change the process definition of a running process instance."
+---
+
+:::danger Experimental feature
+Process instance migration is an experimental feature which is still under development.
+We recommend using it with care and trying out migrations in a test environment before doing so on production.
+
+Pause all interactions with your process instance before migration, including:
+
+- All related job workers
+- Publishing any related messages
+- Triggering timers at the time of migration
+
+Allow the process instance to reach a wait state (its awaiting input) to continue.
+
+:::
+
+Process instance migration fits a running process instance to a different process definition.
+This can be useful when the process definition needs changes due to bugs or updated requirements.
+
+:::tip
+To repair a broken process instance without making changes to the process definition, use [process instance modification](./process-instance-modification.md) instead.
+:::
+
+Use the [migration command](/apis-tools/grpc.md#migrateprocessinstance-rpc) to change the process model of a running process instance.
+
+## Changing the inactive parts of a process instance
+
+Process instance migration allows you to adjust the process model of your process instance to fit to new requirements.
+
+Let's consider an example.
+
+The process below contains a completed start event, an active service task `A`, and an end event. Once you complete the service task `A`, the process will reach the end event.
+
+<!-- TODO: add screenshot of simple example process instance with an active service task `A` -->
+
+However, the requirements of our process have changed.
+
+Instead of completing the process, we want to add a user task `B` after the service task `A` and before the end event.
+
+<!-- TODO: add screenshot of simple example target process with an additional user task `B` after `A` -->
+
+We can create new process instances according to this new process model after deploying it, but we also want our process instance to reach user task `B` when service task `A` is completed.
+
+<!--
+I'm using process, process model, and process definition interchangeably, because I don't know what's best.
```

Comment:
I will create an issue for this to define usage of these terms and adjust the documentation retroactively, as this is likely an issue across more than just this doc.